Linux_scripts/RHUI_repo_validation_scripts/scenarios.py

```diff
@@ -1,99 +1,96 @@
+   logging.info(colors.GREEN + "Repo server IP connectivity is successful" + colors.ENDC)
 
 def msg_rhui_package_check_success():
-   print (colors.GREEN + "rhui-package check is successful" + colors.ENDC)
+   logging.info(colors.GREEN + "rhui-package check is successful" + colors.ENDC)
 
 def msg_rhui_package_check_error():
-   print (colors.RED + "rhui-azure package is not available, Install rhui-azure by downloading manually and rerun this script" + colors.ENDC)
+   logging.error(colors.RED + "rhui-azure package is not available, Install rhui-azure by downloading manually and rerun this script" + colors.ENDC)
    exit()
 
 
 def msg_ip_tables_check():
-   print (colors.RED + "Validate the Reject rules in OUTPUT chain which is listed above and please make sure connectivity to RHUI IPs over 443 is not blocked" + colors.ENDC)   
+   logging.error(colors.RED + "Validate the Reject rules in OUTPUT chain which is listed above and please make sure connectivity to RHUI IPs over 443 is not blocked" + colors.ENDC)
    exit()
 
 def msg_rhui_ip_connectivity_success():
-   print (colors.GREEN + "Repo server IP connectivity is successful" + colors.ENDC)
+   logging.info(colors.GREEN + "Repo server IP connectivity is successful" + colors.ENDC)
 def msg_multiple_yum_process():
-   print (colors.RED + "seems another yum process is running. Please rerun the script once the other yum process completes" + colors.ENDC)
-   print(colors.YELLOW + "if the other yum process was not initiated by you and ifs its not stopping for some reason, you can try killing it using the below commands and then try rerunning the script\n" + colors.ENDC)
-   print (" #ps aux | grep -i yum (prints the process id of all yum running processes)")
-   print (" #kill -9 yum_process_id (kills the process based on provided PID)" + colors.ENDC)
+   logging.error(colors.RED + "seems another yum process is running. Please rerun the script once the other yum process completes" + colors.ENDC)
+   logging.info(colors.GREEN + "RESOLUTION : if the other yum process was not initiated by you and ifs its not stopping for some reason, you can try killing it using the below commands and then try rerunning the script\n   #ps aux | grep -i yum (prints the process id of all yum running processes) \n   #kill -9 yum_process_id (kills the process based on provided PID)" + colors.ENDC)
    exit()
 
 def msg_yum_repolist_success():
-   print(colors.GREEN + "yum check-update is successful. No errors observed" + colors.ENDC)
+   logging.info(colors.GREEN + "yum check-update is successful. No errors observed" + colors.ENDC)
    exit()
 
 def msg_new_scenario():
-   print(colors.GREEN + "This seems to be a new issue and not the regular one, please engage Azure Linux escalation team for further troubleshooting" + colors.ENDC)
+   logging.info(colors.RED + "This seems to be a new issue and not the regular one, please engage Azure Linux escalation team for further troubleshooting" + colors.ENDC)
```

Comment:
add azlinux escalation contact steps

Reply:
Didn't provide azlinux escalation team DL ID as this script will also be executed by customer. But provided suggestion to log support ticket.